cmd/snap/cmd_changes.go

```diff
@@ -198,9 +198,9 @@ func (c *cmdTasks) showChange(chid string) error {
 
 const line = "......................................................................"
 
-func warnMaintenance(cli *client.Client) error {
+func warnMaintenance(cli *client.Client, opName string) error {
 	if maintErr := cli.Maintenance(); maintErr != nil {
-		msg, err := errorToCmdMessage("", maintErr, nil)
+		msg, err := errorToCmdMessage("", opName, maintErr, nil)
```

Comment:
I think here it's fine to pass "" for opName without adding a new argument to the function as the set of possible Maintenance error is limited and actually mostly orthogonal to the operation attempted anyway